internal/services/resource/resource_group_data_source.go

```diff
@@ -50,5 +52,6 @@ func dataSourceResourceGroupRead(d *pluginsdk.ResourceData, meta interface{}) er
 
 	d.Set("name", resp.Name)
 	d.Set("location", location.NormalizeNilable(resp.Location))
+	d.Set("managedBy", *resp.ManagedBy)
```

Comment:
We need to nil check this
```suggestion
	managedBy := ""
	if v := resp.ManagedBy; v != nil {
		managedBy = *v
	}
	d.Set("managedBy", managedBy)
```